junit_tests/DateTimeTest.java

```diff
@@ -0,0 +1,707 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.python.Object;
+import org.python.exceptions.SyntaxError;
+import org.python.exceptions.TypeError;
+import org.python.exceptions.ValueError;
+import org.python.stdlib.datetime.DateTime;
+import org.python.types.Bool;
+import org.python.types.Float;
+import org.python.types.Int;
+import org.python.types.Str;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class DateTimeTest {
+    private DateTime dateTime;
+
+    @BeforeEach
+    public void initDatetime() {
+        dateTime = new DateTime(new org.python.Object[]{
+            Int.getInt(2021),
+            Int.getInt(9),
+            Int.getInt(14),
+            Int.getInt(13),
+            Int.getInt(43),
+            Int.getInt(11),
+            Int.getInt(432523)
+        }, Collections.emptyMap());
+    }
+
+    @Test
+    public void testToday() {
+        DateTime.today(); // TODO: Mock?
```

Comment:
Remaining TODO, you should either finish it or remove it

Reply:
Mocking didn't seem necessary. Remove comment in 946bb34.